.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
```

Comment:
I'm a tab person myself, but this is not the place to have this conversation. This shareable config just follows the Google styleguide.

If you're looking for something with tabs, check out [eslint-config-xo](https://github.com/sindresorhus/eslint-config-xo).